src/utils.ts

```diff
@@ -378,6 +379,20 @@ class Utils {
     isFirefox(): boolean {
         return typeof(browser) !== "undefined";
     }
+
+    getHash(value: string, times=5000): string {
+        if (times <= 0) return "";
+
+        for (let i = 0; i < times; i++) {
```

Comment:
What is the idea behind hashing it 5k times?

And take a look at native function: https://developer.mozilla.org/en-US/docs/Web/API/SubtleCrypto/digest

Reply:
That's a default value taken from the server code 
https://github.com/ajayyy/SponsorBlockServer/blob/master/src/utils/getHash.js#L3-L12

On the server side, userIDs are hashed 5000 times before being stored in the database.

Reply:
It is to be the same as the server, which hashes 5k times for it's operation.

That's a good idea to use the browser crypto function, thanks!

Reply:
But what is the reason for hashing it more than once?

Reply:
It was to make brute forcing harder since the hash of the private userID is in the public database.

Reply:
Fair enough, I didn't spot that it's called with times=1 for video IDs.